README.md

```diff
@@ -24,6 +25,7 @@ nice visual feedback.
 
 * Vim with `+terminal`, or Neovim >= 0.5
 * Under Windows, only Neovim is supported at the moment
+* Target completion requires CMake 3.14 or later
```

Comment:
E.g. "Target autocompletion uses the `cmake-file-api(7)`, which requires CMake 3.14 or later"

Reply:
It's not autocompletion, is it?

Reply:
I think just "completion" is enough